EIPS/eip-1108.md

```diff
@@ -33,12 +33,48 @@ note](https://github.com/ethereum/go-ethereum/pull/16301#issuecomment-372687543)
 the computational cost of `ECADD`, `ECMUL`, and pairing checks (excepting the
 constant) has dropped roughly an order of magnitude across the board.
 
-Also, [optimisations in the bn library](https://github.com/paritytech/bn/pull/9) 
+Also, optimizations in the bn library [in 2018](https://github.com/paritytech/bn/pull/9) and [2019](https://github.com/paritytech/bn/pull/14)
 used by the [Parity client](https://github.com/paritytech/parity-ethereum) led to a 
 significant performance boost we 
 [benchmarked](https://gist.github.com/pdyraga/4649b74436940a01e8221d85e80bfeef) 
 and compared against the [previous 
-results](https://github.com/ethereum/benchmarking/blob/master/constantinople/analysis2.md). 
+results](https://gist.github.com/zac-williamson/838410a3da179d47d31b25b586c15e53).  
+
+## The Rationale Behind Lowering Gas Costs  
+
+### Existing protocols would benefit immensely from cheaper elliptic curve cryptography
+
+Fast elliptic curve cryptography is a keystone of a growing number of protocols built on top of Ethereum. To list a few:  
+
+  [The AZTEC protocol](https://github.com/AztecProtocol/AZTEC) utilizes the elliptic curve precompiles to construct private tokens, with zero-knowledge transaction logic, via the [ERC1723](https://github.com/ethereum/EIPs/issues/1723) and [ERC1724](https://github.com/ethereum/EIPs/issues/1724) standard.  
```

Comment:
Can you make these into a bullet point list? It would look nicer that way after formatting.

Reply:
Done